src/components/views/userProfile/projectsTab/ProjectVerificationStatus.tsx

```
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import {
+	SublineBold,
+	brandColors,
```

ESLint

[error] 'brandColors' is defined but never used.

(unused-imports/no-unused-imports)